### ISSUE TITLE
Bug Report: Tooltip Not Visible When accessibility.enabled is true

### ISSUE THREAD

**Issue by amit-tsubery-sp** [highcharts/highcharts#22397]
Description:
When accessibility.enabled is set to true in a pie chart configuration, tooltips are not visible for legend items with long text. However, when accessibility.enabled is set to false, the tooltips appear as expected.

Steps to Reproduce:

- Open the provided [JSFiddle example](https://jsfiddle.net/jfuvskxo/4/).
- Observe the tooltip behavior with accessibility.enabled set to false (current state).
- Change accessibility.enabled to true in the chart configuration.
- Hover over a long legend item and notice that the tooltip is no longer displayed.

Expected Behavior:
The tooltip should be visible when hovering over long legend items, regardless of the accessibility.enabled setting.

Actual Behavior:
The tooltip is not visible when accessibility.enabled is true.

Live Example:
[JSFiddle link](https://jsfiddle.net/jfuvskxo/4/)
(Currently, accessibility.enabled is false. Change it to true to reproduce the issue.)

**Comment by TorsteinHonsi** [comment]
Thanks for reporting!

Hint for the developer fixing this: The `title` attribute from the SVG text should be copied over to the shim button that a11y inserts.
![Skjermbilde 2024-12-18 kl  13 04 15](https://github.com/user-attachments/assets/133d4831-f2cf-4b5f-8cce-4368a553701c)

**Comment by TorsteinHonsi** [comment]
Note: "tooltip" in this context is not the Highcharts tooltip, but the native browser tooltip. 

![Image](https://github.com/user-attachments/assets/0953740d-b6fb-47af-92ef-0267b7eacd35)
